users/internal/handlers/users.go

```diff
@@ -36,6 +37,7 @@ func (u Users) Routes() http.Handler {
 
 	r.Post("/", u.Create)
 	r.Get("/{id}", u.GetOne)
+	r.Patch("/{id}", u.UpdateUserName)
```

Comment:
I would recommend to read this https://restfulapi.net/ the first 3 articles in the section 'Learn'